src/vivarium_census_prl_synth_pop/components/household.py

```diff
@@ -227,22 +243,3 @@ def generate_po_boxes(self, num_addresses) -> np.array:
             additional_key="po_box_number",
         )
         return np.array(po_boxes)
-
-    def randomly_sample_states_pumas(
```

Comment:
NOTE: Since this now gets called by two different components (household and business), I moved it into utilities.py. It's very slightly more than a copy/paste job, though, so I'd recommend reviewing the function there.